tests/test_update_lockfile.py

```diff
@@ -53,7 +53,7 @@ def test_update_lockfile_minimal_python(workdir, monkeypatch):
         lockfile_content = f.read()
```

Comment:
I'd make a comment here about 3.6 ecosystem being unreliable wrt to package normalization and do a `lockfile_content = lockfile_content.replace("_", "-")`